@ecomplus/storefront-components/src/html/ShippingCalculator.html

```diff
@@ -70,6 +70,7 @@
               <shipping-line
                 :shipping-line="service.shipping_line"
                 :production-deadline="productionDeadline"
+                :service-code="service.service_code"
```

Comment:
Por que não:

```suggestion
                :data-service-code="service.service_code"
```

?

E aí não precisa adicionar a prop, nem alterar mais nada na verdade...

Reply:
Oia, pensei que não dava pra fazer essa inserção direta. Melhor ainda. Valeu